src/database/migrations/2022_03_01_145900_change_grants_table_columns.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeGrantsTableColumns extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
Farò un commento impopolare, perché, anche se non sembra, io sono molto buona :laughing: 
Io rinominerei la tabella `grants` in `grants_backup` prima di operare violentemente sulle colonne.. Mi sembra un gesto carino nei confronti degli utenti. Ma penso che nessuno sarà d'accordo!

Reply:
Va bene cosi non perdiamo i dati :dagger: